# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../syslog-ng/head_n.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur syslog-ng-1.9.4+20050701.orig/configure.in syslog-ng-1.9.4+20050701/configure.in
--- syslog-ng-1.9.4+20050701.orig/configure.in	2005-07-01 00:11:05.000000000 +0200
+++ syslog-ng-1.9.4+20050701/configure.in	2005-07-01 20:21:36.000000000 +0200
@@ -58,7 +58,7 @@
 AC_PROG_RANLIB
 
 AC_MSG_CHECKING(how to enable static linking for certain libraries)
-ldversion=`ld -V 2>&1 | head -1`
+ldversion=`ld -V 2>&1 | head -n 1`
 if echo $ldversion | egrep "GNU|Solaris" > /dev/null; then
         LD_START_STATIC="-Wl,-Bstatic"
         LD_END_STATIC="-Wl,-Bdynamic"
